docs/take-to-prod.md

```diff
@@ -0,0 +1,85 @@
+Taking your .NET for Apache Spark Application to Production
+===
+
+# Table of Contents
+This how-to provides general instructions on how to take your .NET for Apache Spark application to production.
+In this documentation, we will summarize the most commonly asked scenarios when running a .NET for Apache Spark Application.
+You will also learn how to package your application and submit your application with [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) and [Apachy Livy](https://livy.incubator.apache.org/).
+- [How to deploy your application when you have a single dependency](#how-to-deploy-your-application-when-you-have-a-single-dependency)
+  - [Scenarios](#scenarios)
+  - [Package your application](#package-your-application)
+  - [Launch your application](#launch-your-application)
+- [How to deploy your application when you have multiple dependencies](#how-to-deploy-your-application-when-you-have-multiple-dependencies)
+  - [Scenarios](#scenarios-1)
+  - [Package your application](#package-your-application-1)
+  - [Launch your application](#launch-your-application-1)
+
+## How to deploy your application when you have a single dependency
+### Scenarios
+#### 1. SparkSession code and business logic in the same Program.cs file
+This would be the simple use case when you have `SparkSession` code and business logic (UDFs) in the same Program.cs file and in the same project (e.g. mySparkApp.csproj).
```

Comment:
```suggestion
The `SparkSession` code and business logic (UDFs) are contained in the same `Program.cs` file and in the same project (e.g. mySparkApp.csproj).
```
If they are contained in the same Program.cs, then by definition wouldn't it be in the same project ?  I don't think you need to say "in the same project"

Reply:
I agree, I have removed "in the same project". Thanks!